qa/automation/CareerPortal.Tests.Scenarios/Send your CV/Send_your_CV.feature

```diff
@@ -0,0 +1,108 @@
+Feature: 1.1.2 Send_CV.feature
+
+Background: 
+	Given user accessed Quantori Career Portal from his browser
+	And user scrolled down to CV sending form
+
+Scenario: 1.1.2_01 User can successfully send CV
```

Comment:
I guess, that you need to add a check that the user can submit the form, leaving the "Message" field empty)